### ISSUE TITLE
sys.stdin as UTF-8

### ISSUE THREAD

**Issue by shidocchi** [shidocchi/text2docx#1]
I use this on Cygwin: 

```
$ echo '日本語' | python -m text2docx
```

But any UTF-8 Japanese text was not accepted:

```
$ file sample.txt
sample.txt: Unicode text, UTF-8 text

$ cat sample.txt | python -m text2docx
(...)
UnicodeEncodeError: 'utf-8' codec can't encode character '\udc87' in position 45: surrogates not allowed
```

sys.stdin represents UTF-8 encoding but CP932 encoding is accepted:

```
$ cat sample.txt | nkf -s | python -m text2docx
```

**Comment by shidocchi** [comment]
I think this UnicodeEncodeError occurs only Windows.
What about Linux occurs this error?

I will try to install on Linux vm later.

**Comment by shidocchi** [comment]
TextIOWrapper reconstruction seems to work better.

```
sys.stdin = io.TextIOWrapper(sys.stdin.buffer encoding='utf-8')
```

But maybe in future it is like to bothers us.
I choose to add new arg which can remove this treatment.

**Comment by shidocchi** [comment]
merged #4
